modules/nodes-nodes-viewing-listing.adoc

```diff
@@ -59,9 +59,9 @@ $ oc get nodes -o wide
 [source,terminal]
 ----
 NAME                STATUS   ROLES    AGE    VERSION   INTERNAL-IP    EXTERNAL-IP   OS-IMAGE                                                       KERNEL-VERSION                 CONTAINER-RUNTIME
-master.example.com  Ready    master   171m   v1.22.0   10.0.129.108   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.0-30.rhaos4.8.gitf2f339d.el8-dev
-node1.example.com   Ready    worker   72m    v1.22.0   10.0.129.222   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.0-30.rhaos4.8.gitf2f339d.el8-dev
-node2.example.com   Ready    worker   164m   v1.22.0   10.0.142.150   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.0-30.rhaos4.8.gitf2f339d.el8-dev
+master.example.com  Ready    master   171m   v1.22.1   10.0.129.108   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
+node1.example.com   Ready    worker   72m    v1.22.1   10.0.129.222   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
+node2.example.com   Ready    worker   164m   v1.22.1   10.0.142.150   <none>        Red Hat Enterprise Linux CoreOS 48.83.202103210901-0 (Ootpa)   4.18.0-240.15.1.el8_3.x86_64   cri-o://1.22.1-30.rhaos4.8.gitf2f339d.el8-dev
```

Comment:
My expectation is that other values change, but unlikely to be a customer concern.